src/components/IconTable/index.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { css } from 'emotion';
+
+interface Icon {
+  src: string;
+  left: number;
+  top: number;
+  width: number;
+  height: number;
+  onMouseEnterHandler?: (event?: React.MouseEvent) => void;
+  onMouseExitHandler?: (event?: React.MouseEvent) => void;
+  onClickHandler?: (event?: React.MouseEvent) => void;
+}
+
+interface IconTableProps {
+  heightVW: number;
+  backgroundSrc: string;
+  icons: Icon[];
+}
+
+export default class IconTable extends React.Component<IconTableProps> {
+  constructor(props) {
+    super(props);
+  }
+
+  drawIcons() {
+    const {icons} = this.props;
+
+    return icons.map(({
+      src, left, top, width, height,
```

Comment:
I wouldn’t destructure this many things; just have a variable called icon and access icon.height etc in your map

Reply:
(Lol I guess I was not paying too much attention to coding style... let’s fix it in another PR)